### DISCUSSION TITLE
versione 1.25

### DISCUSSION THREAD

**Discussion by annamaraspin** [italia/ansc#737] [Q&A] [answered]
E' possibile avere il changelog.md aggiornato con le modifiche ?
grazie

**Comment by mttfranci** [comment, marked as answer]
@annamaraspin oltre al [Changelog](https://github.com/italia/ansc/blob/main/docs/Changelog.md) è possibile fare riferimento alle note dei rilasci : 
<https://github.com/italia/ansc/releases>
Ad esermpio nel caso della versione 1.25 : 
<https://github.com/italia/ansc/releases/tag/v1.25.0>

Saluti

Assistenza ANSC

**Comment by annamaraspin** [comment]
@mttfranci 
Grazie, avevo chiuso il ticket poco dopo averlo scritto perché avevo trovato le modifiche

**Reply by mttfranci** [reply]
Di niente, lo abbiamo immaginato, abbiamo deciso comunque di rispondere qualora possa essere utile ad altri.

Saluti

Assitenza ANSC
